### DISCUSSION TITLE
NYT paywall slider obscure

### DISCUSSION THREAD

**Discussion by jo99jo** [uBlockOrigin/uAssets#19552] [Q&A]
When I go to the NYT website in Edge and have run past their free article limit, an obnoxious slider runs upward prompting to pay for further contact.

I understand that UBlock isn't for removing paywalls.  

However, this slider can be removed using the UBlock zapper.  I used to be able to remove this annoyance with a single click, now it requires two.  

I posed this question previously here: https://github.com/uBlockOrigin/uAssets/issues/18858 

and someone suggested using the element picker to create a permanent rule that blocks this slider.  

But the element pickup doesn't appear to work for me on the NYT website.  All I get is a "+" sign but no ability to do anything else, like a create a rule.

The element picker appears to work for this person but does not work for me.  Can anyone tell me what I might be doing wrong so that the element picker doesn't work correctly?

![UBlock - NYT - Element pickup mode](https://github.com/uBlockOrigin/uAssets/assets/48929420/385d35e4-5f0e-447c-9112-dcbb5ab5d7d9)

**Comment by ghost** [comment]
Make sure cosmetic filtering is enabled.

**Comment by uBlock-user** [comment]
```
www.nytimes.com###gateway-content
||meter-svc.nytimes.com^
||mwcm.nytimes.com^
||samizdat-graphql.nytimes.com^
```
